packages/web-client/src/helpers/space/functions.ts

```diff
@@ -169,20 +169,20 @@ export function buildSpace(data): SpaceResource {
     canDownload: function () {
       return true
     },
-    canBeDeleted: function ({ user }: { user?: User } = {}) {
-      return this.disabled && (user?.role?.name === 'spaceadmin' || this.isManager(user))
+    canBeDeleted: function ({ user, ability }: { user?: User; ability?: any } = {}) {
+      return this.disabled && (ability?.can('delete', 'Space') || this.isManager(user))
     },
     canRename: function ({ user }: { user?: User } = {}) {
-      return user?.role?.name === 'spaceadmin' || this.isManager(user)
+      return this.isManager(user)
```

Comment:
Why did you decide to only remove the role/permission check from some of the functions?

Reply:
Because editing is apparently not supported for space admins (except when they are managers of the space). At least that's currently the case...

I'm still a bit confused by the permissions the backend sends, and there are also bugs. We might re-introduce that later. But I don't know how that's going so look like, so I decided to kill it for now.

Reply:
ok, thanks for the input 👍